packages/python-packages/protocol-stub-generator/protocol/protocol_models.py

```diff
@@ -765,16 +773,16 @@ def get_type(data, page=False):
                 return_type = data["elementType"]["language"]["default"]["name"] + "[]"
             else:
                 return_type = data["elementType"]["language"]["default"]["name"]
-        if return_type == "number":
+        if "number" in return_type:
```

Comment:
judging by the code above, `return_type` is a string, but here it looks like we're expecting an iterable. I guess I'm wondering why the change? Should the others be changed too?

Reply:
this was to catch if number was in any part of the return to replace it

Reply:
discussed offline